pkg/apis/hub/v1alpha1/api_version.go

```diff
@@ -45,9 +45,6 @@ type APIVersion struct {
 
```

Comment:
The `APIName`, `ServiceName` and `ServicePort` additional printer columns can be removed.